src/App.tsx

```diff
@@ -1,156 +1,81 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useCallback, useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Notification } from './components/Notification';
+import { TodoList } from './components/TodoList';
+import { ErrorMessages } from './types/ErrorMessages';
+import { FilterBy } from './types/FilterBy';
+import { Todo } from './types/Todo';
+
+const USER_ID = 6315;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(FilterBy.ALL);
+  const [hasError, setHasError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(ErrorMessages.NONE);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(fetchTodos => {
+        setTodos(fetchTodos);
+      })
+      .catch(() => {
+        setHasError(true);
+        setErrorMessage(ErrorMessages.ONLOAD);
+      });
+  }, []);
+
+  const visibleTodos = todos.filter(todo => {
```

Comment:
all additional functionality that you used for your data will be better to move to separate utils function instead of writing this code directly in the component